sqe-database-access/Queries/Text.cs

```diff
@@ -9,98 +9,191 @@ public static class GetTextChunk
         ///     @editionId is the Id of the edition the text is to be taken from
         /// </summary>
         public const string GetQuery = @"
```

Comment:
This is a huge query. I suspect it will cause major performance issues. It's worth running it through a query analyzer (I'm sure MariaDB has one) to see which indices are used and how long it takes to execute.

Reply:
Sorry, I made a mistake and did not push the latest commit from this branch up to GitHub.  You will see a cleaner query and, best of all, the tests pass. I did run some query analyses, and the query runs pretty well, but I still have some worries about the time to complete.

I am still thinking of preprocessing the JSON objects for these text queries on each update to the edition, but I want some better usage stats before I get into that.